authcrypt/main.cpp

```diff
@@ -24,8 +24,13 @@
 #include "mbedtls/platform.h"
 
 int main() {
+    mbedtls_platform_context platform_ctx;
+    if(mbedtls_platform_setup(&platform_ctx) != 0) {
```

Comment:
I suggest not covering up the return code here and also adding something like:
```
printf( "Platform initialization failed with error -%04X",  -ret );
```
Not sure what the format should be but thats how we normally print error codes.